docs/section5/Rule5-53_nonprimary.md

```diff
@@ -0,0 +1,60 @@
+# Envelope - Rule 5-53  
+**Schema Version** 0.0.23
+**Primary Rule** True
+**Rule ID:** 5-53  
+**Rule Description:** U-factor of the baseline door is based on Tables G3.4-1 through G3.4-8 for the applicable door type (swinging or non-swinging) and envelope conditioning category.
+**Appendix G Section:** Section 5 Envelope  
+**Appendix G Section Reference:** Section G3.1-5(b) Building Envelope Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**
+  1. The surface has an opaque door subsurface
+
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** Tables G3.4-1 to G3.4-8  
+**Function Call:**  
+
+  1. get_surface_conditioning_category()
+
+## Rule Logic:  
+
+- Get building climate zone: ```climate_zone = B_RMR.weather.climate_zone```
+
+- Get surface conditioning category dictionary for B_RMR: ```scc_dictionary_b = get_surface_conditioning_category(B_RMR)```
+
+- For each building segment in the Proposed model: ```for building_segment_b in B_RMR.building.building_segments:```
+
+  - For each zone in building_segment: ```for zone_b in building_segment_b.zones:```
+
+    - For each surface in zone: ```for surface_b in zone_b.surfaces:```
+
+      - Get surface conditioning category: ```scc_b = scc_dictionary_b[surface_b.id]```
+
+      - For each subsurface in surface: ```for subsurface_b in surface_b.subsurfaces:```
+
+        - Check if subsurface is opaque door: ```if ( subsurface_b.classification == "DOOR" ) AND ( subsurface_b.glazed_area < subsurface.opaque_area )```
```

Comment:
I beleive it should be "subsurface_b.glazed_area <= subsurface.opaque_area" based on Section 3 deifinition.